### PR TITLE
Prepare v2

### PR DESCRIPTION
**Pull request by apfelbox**

### CODE REVIEW

File: src/Hosting/HostingEnvironment.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Hosting\Hosting;
+
+use Torr\Hosting\Tier\HostingTier;
+
+final class HostingEnvironment
```
Comment (keichinger) [inline src/Hosting/HostingEnvironment.php:7]: Proposal: What do you think about the name `DeploymentEnvironment`?

File: src/DependencyInjection/HostingBundleConfiguration.php [hunk 2]
```diff
@@ -18,8 +18,7 @@ public function getConfigTreeBuilder()
 		$treeBuilder->getRootNode()
 			->children()
 				->enumNode("tier")
-					->isRequired()
-					->values(HostingTier::getAllowedTiers())
+					->values(HostingTier::getAllowedConfigValues())
```
Comment (keichinger) [inline src/DependencyInjection/HostingBundleConfiguration.php:21]: If only Symfony 6.2 was already here :D https://symfony.com/blog/new-in-symfony-6-2-improved-enum-support

File: src/Hosting/HostingEnvironment.php (hunk 1 above)
Comment (apfelbox) [reply src/Hosting/HostingEnvironment.php:7]: Would  also work, but as this bundle is called `21torr/hosting` and not `21torr/deployment` it should stay as is imo

File: src/DependencyInjection/HostingBundleConfiguration.php (hunk 2 above)
Comment (apfelbox) [reply src/DependencyInjection/HostingBundleConfiguration.php:21]: I don’t think, that they support enums in config nodes, even if they are called enumNode. 

It at least is not mentioned in the blog post.

**Pull request merged**